integration-tests/api/__tests__/admin/shipping-profile.spec.ts

```diff
@@ -84,6 +84,17 @@ medusaIntegrationTestRunner({
             created_at: expect.any(String),
           })
         )
+
+        const { data } = await api.delete(
```

Comment:
suggestion: Let's attempt to retrieve the shipping profile after to ensure this is not a false positive.